test/units/testsuite-21.sh

```diff
@@ -21,16 +21,25 @@ at_exit() {
     fi
 }
 
+add_suppression() {
+    local interface="${1:?}"
+    local suppression="${2:?}"
+
+    sed -i "/\[$interface\]/a${suppression//\//\\\/}" /etc/dfuzzer.conf
```

Comment:
Hmm, why not switch to a different separator? Say `#` or `|`

Reply:
That works only in the `s` sed command.

Reply:
`sed -i "\|\[$interface\]| a${suppression}" /etc/dfuzzer.conf` works for me?

Reply:
(The key is to escape the starting separator)

Reply:
Ah, interesting:

```
'\%REGEXP%'
     (The '%' may be replaced by any other single character.)

     This also matches the regular expression REGEXP, but allows one to
     use a different delimiter than '/'.  This is particularly useful if
     the REGEXP itself contains a lot of slashes, since it avoids the
     tedious escaping of every '/'.  If REGEXP itself includes any
     delimiter characters, each must be escaped by a backslash ('\').
```

didn't know about this syntax, thanks!